dynamic-plugins/wrappers/backstage-plugin-catalog-backend-module-bitbucket-server/dist-dynamic/yarn.lock

```diff
@@ -34,9 +34,9 @@
     form-data "^4.0.0"
 
 "@types/node@*":
-  version "20.12.11"
-  resolved "https://registry.yarnpkg.com/@types/node/-/node-20.12.11.tgz#c4ef00d3507000d17690643278a60dc55a9dc9be"
-  integrity sha512-vDg9PZ/zi+Nqp6boSOT7plNuthRugEKixDv5sFTIpkE89MmNtEArAShI4mxuX2+UrLEe9pxC1vm2cjm9YlWbJw==
+  version "20.14.1"
```

Comment:
this is newer than 20.12, which is in the RPMs in the ubi9/nodejs-20 container we use to build. Is this safe?

Reply:
yes